changes/fix-poison-crash.md

```diff
@@ -0,0 +1 @@
+Fixed a crash that can happen when an explosive monster dies from poison.
```

Comment:
This applies to more cases than poison, turns out.

Reply:
Thanks for verifying that the new crash is also fixed by this!

Updated this file and the commit description.